src/App.js

```diff
@@ -1,6 +1,90 @@
-import React from 'react';
+import React, { PureComponent } from 'react';
 import './App.css';
+import Pagination from './components/Pagination/Pagination';
 
-const App = () => <h1>Pagination</h1>;
+const optionToChoseItemsCountPerPage = [3, 5, 10, 20];
+
+class App extends PureComponent {
+  state = {
+    currentPage: 1,
+    itemsPerPage: 5,
+    totalItems: 42,
+  }
+
+  changePage = (page) => {
```

Comment:
consider creating separate functions for changing pages with buttons 'prev' and 'next' and by clicking on the page number

Reply:
The first one will just increment or decrement current page number in state and the second one will set a new page number to state